audit/models.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class AuditLog(models.Model):
+    CREATE = 'c'
+    UPDATE = 'u'
+    DELETE = 'd'
+
+    ACTION_CHOICES = (
+        (CREATE, "Create"),
+        (UPDATE, "Update"),
+        (DELETE, "Delete")
+    )
+
+    action_at = models.DateTimeField(auto_now_add=True)
+    action_by = models.ForeignKey(User, related_name='auditlog_action_by',
+                                  null=False, blank=False)
```

Comment:
I don't quite remember the data model for the CI service, just want to check that you've checked that every user that can log into the CI will also have a user on the CI service.

Reply:
@rudigiesler I see now that the CI Service token is configured as a env var so I can't use `self.request.user.id` to get the correct user.

I'll need to investigate a bit further here.

Thanks for picking this up.